File: repos/DS_assignment2p2_2023/DS_assignment2p2_2023/gamedriver.cpp
#include "battleship.h"
#include <iostream>
int main() {
	Fleet fl;
	Location loc;
	fl.deployFleet();
	char check;
	while (fl.operational()) {
		cout << "would you like to print the fleet [Y/N]"; cin >> check;
		if (check == 'y' || check == 'Y') {
			fl.printFleet(); 
			loc.fire();
			if (fl.isHitNSink(loc) == 1) {
				loc.print();
				cout << "hit" << endl;
			}
			else if (fl.isHitNSink(loc) == 0) {
				loc.print();
				cout << "miss" << endl;
			} 
		} 
		else {
			loc.fire();
			if (fl.isHitNSink(loc) == 1) {
				loc.print();
				cout << "hit"<<endl; 
			}
			else if (fl.isHitNSink(loc) == 0) {
				loc.print();
				cout << "miss"<<endl; 
			}
		}
	}
}

File: eclipse-workspace/cs2assignment4/src/cs2assignment4/gameDriver.java
package cs2assignment4;

public class gameDriver {

	public static void main(String[] args) {
		//testing location class
Location location=new Location();
Fleet fleet=new Fleet();
//location.print();
//location.pick();
//location.print();
//location.fire();
//location.print();
//System.out.println("\n------ ending location test --------");
//
//		//ship testing
//Ship ship=new Ship();
//ship.printShip();
//
//ship.setLocation(location);
//
//ship.printShip();
//
//System.out.println(ship.isSunk());
////ship.sink();
////ship.printShip();
//
////test fleet class 
//System.out.println("------------");
fleet.deployFleet();


while(fleet.operational()) {
	fleet.printFleet();
	location.fire();
	fleet.check(location);
	fleet.isHitNSink(location);
	
	

}



	}
	

}
